androidApp/src/main/java/dev/johnoreilly/confetti/sessions/SessionListView.kt

```diff
@@ -201,33 +201,37 @@ fun SessionView(
         val user by remember { mutableStateOf(authentication.currentUser()) }
         var showDialog by remember { mutableStateOf(false) }
 
-        if (isBookmarked) {
-            Icon(
-                imageVector = Icons.Outlined.Bookmark,
-                contentDescription = "remove bookmark",
-                tint = MaterialTheme.colorScheme.primary,
-                modifier = Modifier
-                    .clickable {
-                        if (user != null) {
-                            removeBookmark(session.id)
-                        } else {
-                            showDialog = true
+        when(isBookmarked) {
```

Comment:
Hmm, I don't remember doing this. Maybe I did a mistake during the conflict resolution? I will double check.

Reply:
Changes reverted.